### DISCUSSION TITLE
Filter single samples differently

### DISCUSSION THREAD

**Discussion by Desi0** [GreenleafLab/ArchR#789] [Questions / Documentation]
Looks like I need different filtering conditions for some of my samples. How can I do that? When creating the Arrow files, all samples are treated the same way. Therefore, I wanted to add a more stringent filtering afterwards for these samples, but I cannot manage to do so.
Can somebody help me?
Thanks

**Comment by rcorces** [comment]
You can create arrow files individually if you would like to apply different filters to different samples. After creation, you can group all arrow files together into a single ArchRProject.

**Reply by Desi0** [reply]
Good point. I will try this. Thanks for the fast help
